jupyter_elfinder/elfinder.py

```diff
@@ -368,20 +411,21 @@ def run(
                 }
 
         if self._error_data:
+            # TODO: Is errorData a valid response item?  # pylint: disable=fixme
             self._response["errorData"] = self._error_data
```

Comment:
Where do we use `"errorData"` in the response?

Reply:
It's some kind of left over from the previous version. Indeed, no where is using it.

I think we can move that to the `self.__debug`, say, `self.__debug('errorData', self._error_data)`.